### ISSUE TITLE
[Suggestion] Please add an option to skip starting segments together with --hls-prefer-native

### ISSUE THREAD

**Issue by whyau** [rg3/youtube-dl#6058]
Downloading seems to be segment-by-segment with the experimental --hls-prefer-native option, it could be very convenient if there is an additional option to control over the starting segment number so that one can choose to record partial footage from a lengthy youtube video.

**Comment by gabrieleliot** [comment]
I'm going to second this request. I frequently download Twitch streams from Smitegame and the streams will sometimes be 72 hours long. Since the files are stored in small segments, it should be possible to specify a chunk of time to download, e.g. a 4 hour section. Downloading a 3 days stream is silly.

**Comment by whyau** [comment]
There may be a lot of work if we want to specify the chunk of time to download (or start time and stop time), since this may involve parsing of the segment information.  On the other hand, having control over the starting and ending segment numbers could be straight-forward, and I have already done this on my own.

**Comment by gabrieleliot** [comment]
Yes, starting and ending segments would be fine. How have you done this?

**Comment by shirishag75** [comment]
looking forward to know more on this.
